### PR TITLE
Update the return function regex

### PR DESCRIPTION
**Pull request by ahmadMsaleem**
### Description

There was a need for cleaner return functions instead of the old one, because the `::` is so confusing for people who learn by looking at codes

``function getScore(player: player) :: number: `` old 
``function getScore(player: player) returns number:`` new 

---
**Target Minecraft Versions:** 
any
**Requirements:** 
**Related Issues:**

### CODE REVIEW

**Review by TheLimeGlass — APPROVED** [review]

**Review by sovdeeth — APPROVED** [review]

**Review by AyhamAl-Ali — APPROVED** [review]
I personally like this addition, idk why we never did it?! hopfully it doesn't break anything we didn't think of.

Can you update examples to add 1 more example using this new addition.

**Comment by ahmadMsaleem** [comment]
> I personally like this addition, idk why we never did it?! hopfully it doesn't break anything we didn't think of.
> 
> Can you update examples to add 1 more example using this new addition.

The code for StructFunction.java and EffReturn.java have been updated with two new examples

File: src/main/java/ch/njol/skript/structures/StructFunction.java [hunk 1]
```diff
@@ -48,7 +48,10 @@
 	"\tbroadcast {_message} # our message argument is available in '{_message}'",
 	"",
 	"local function giveApple(amount: number) :: item:",
-	"\treturn {_amount} of apple"
+	"\treturn {_amount} of apple",
+	"",
+	"function getPoints(p: player) returns object:",
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/structures/StructFunction.java:53]: Better
```suggestion
	"function getPoints(p: player) returns number:",
```

**Review by AyhamAl-Ali — APPROVED** [review]

**Comment by ahmadMsaleem** [comment]
After testing this for an hour, everything seems fine. Is there anything else I should test?
```
function itemTest1() returns item:
	return test2()

function itemTest2() returns item:
	return test3()

function itemTest3() :: item:
	return diamond sword
```

File: src/main/java/ch/njol/skript/effects/EffReturn.java [hunk 2]
```diff
@@ -43,7 +43,10 @@
 @Description("Makes a function return a value")
 @Examples({
 	"function double(i: number) :: number:",
-		"\treturn 2 * {_i}"
+		"\treturn 2 * {_i}",
+	"",
+	"function divide(i: number) :: number:",
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/effects/EffReturn.java:48]:
```suggestion
	"function divide(i: number) returns number:",
```

**Review by APickledWalrus — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffReturn.java [hunk 3]
```diff
@@ -43,7 +43,10 @@
 @Description("Makes a function return a value")
 @Examples({
 	"function double(i: number) :: number:",
-		"\treturn 2 * {_i}"
+		"\treturn 2 * {_i}",
+	"",
+	"function divide(i: number) returns number:",
+		"\treturn {_i} / 2"
 })
 @Since("2.2")
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/effects/EffReturn.java:51]:
```suggestion
@Since("2.2, INSERT VERSION (returns aliases)")
```

**Review by TheLimeGlass — APPROVED** [review]

**Pull request merged**